layout/_third-party/analytics/index.swig

```diff
@@ -1,3 +1,4 @@
+{% include 'busuanzi-counter.swig' %}
```

Comment:
Also, if u move `busuanzi-counter` index to widget, u must delete it from index, right?